src/oas.md

```diff
@@ -84,6 +84,130 @@ Some examples of possible media type definitions:
   application/vnd.github.v3.patch
 ```
 
+#### Media Type Registry
+
+While the [Schema Object](#schema-object) is designed to describe and validate JSON, several other media types are commonly used in APIs.
+Requirements regarding support for other media types are documented in this Media Types section and in several Object sections later in this specification.
+For convenience and future extensibility, these are cataloged in the OpenAPI Initiative's [Media Type Registry](https://spec.openapis.org/registry/media-type/), which indicates where in this specification the relevant requirements can be found.
+
+#### Sequential Media Types
+
+Several media types exist to transport a sequence of values, separated by some delimiter, either as a single document or as multiple documents representing chunks of a logical stream.
+Depending on the media type, the values could either be in another existing format such as JSON, or in a custom format specific to the sequential media type.
+
+Implementations MUST support modeling sequential media types with the [Schema Object](#schema-object) by treating the sequence as an array with the same items and ordering as the sequence.
```

Comment:
This wording is confusing to me, and doesn't seem to reflect the requirement that the Schema Object modeling the sequence must itself be of `type: array`.

Reply:
There is no requirement that the Schema Object include `type: array`, although it would be a good practice.

What we're talking about here is not so much what to put in the Schema Object, but what data structure to convert the document to in order to use that data structure with the document.

Implementations don't get that from the Schema Object, they get that from these requirements, so it would be an error on the part of the implementation to pass anything but an array here.  Of course, it's good practice to put the `type: array` in, and if you have other tools that depend on the `type` keyword and aren't paying attention to the  media type with which the Schema Object is used, then you have to do that.  But there's no requirement for it to be in the Schema Object.

Reply:
@duncanbeevers my most recent commit (after a force-push that was a re-base of the unchanged original commit) added some clarification here, please see if that helps!